File: PRACTICA/Java/irrgarten/LabyrinthSquare.java
package irrgarten;

public class LabyrinthSquare {
    private int row;
    private int col;
    private char content;

    public LabyrinthSquare(int row, int col, char content) {
        this.row = row;
        this.col = col;
        this.content = content;
    }

    public char get() {
        return content;
    }

    public void set(int row, int col, char content) {
        this.row = row;
        this.col = col;
        this.content = content;
    }
}

File: PRACTICA/Ruby/Irrgarten/labyrinth_square.rb
class Labyrinth_square
    def initialize(row, col, content)
        @row = row
        @col = col
        @content = content
    end

    def get()
        @content
    end

    def set(row, col, content)
        @row = row
        @col = col
        @content = content
    end
end